src/util/Types.ts

```diff
@@ -23,17 +23,16 @@ export type ChannelResolvable = Channel | string;
 export type UserResolvable = User | ClientUser | BaseUser | string;
 export type EventResolvable = 'ready' | 'waitingForGuilds' | 'guildAvailable' | 'guildCreate' | 'guildUpdate' | 'message';
 export type ActivityStatus = 'online' | 'idle' | 'dnd' | 'invisible';
-
 export type MessageContent = string | Embed | {
     content?: string;
-    embeds?: [ Embed ],
+    embeds?: Partial<Embed>[],
```

Comment:
```suggestion
    embeds?: Embed[],
```
The partial type makes no sense here as it makes the whole object optional.

Reply:
It does actually, as all properties are not required. In the options, you'll have to supply `null`s, which is annoying.

Reply:
You'll have to supply 20+ `null`s, as you can see here:
![image](https://user-images.githubusercontent.com/67689562/146951759-12497c8d-05cf-42b9-b1c4-92ec1cc84969.png)

Reply:
> You'll have to supply 20+ `null`s, as you can see here:
> 
> ![image](https://user-images.githubusercontent.com/67689562/146951759-12497c8d-05cf-42b9-b1c4-92ec1cc84969.png)

The embed typings are wrong. I'll approve with `Partial` for now but embeds need certain items like `title`.